// ng2-timeline-simple/dist/ng2-timeline-simple.umd.js
(function (global, factory) {
	typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@angular/core'), require('@angular/common')) :
	typeof define === 'function' && define.amd ? define(['exports', '@angular/core', '@angular/common'], factory) :
	(factory((global['ng2-timeline-simple'] = {}),global._angular_core,global._angular_common));
}(this, (function (exports,_angular_core,_angular_common) { 'use strict';

/**
 * Created by mohma on 7/31/2017.
 */
var Ng2TimelineSimpleComponent = (function () {
    function Ng2TimelineSimpleComponent() {
    }
    /**
     * @return {?}
     */
    Ng2TimelineSimpleComponent.prototype.ngOnInit = function () {
    };
    return Ng2TimelineSimpleComponent;
}());
Ng2TimelineSimpleComponent.decorators = [
    { type: _angular_core.Component, args: [{
                selector: 'ng2-timeline-simple',
                template: "<ul class=\"timeline\"> <li *ngFor=\"let item of items; index as i\"> <div class=\"timeline-badge\" [innerHTML]=\"item.icon\" [ngStyle]=\"{'background-color': item.color}\"> </div> <div class=\"timeline-panel\"> <div class=\"timeline-heading\"> <h4 class=\"timeline-title\">{{item.title}}</h4> </div> <div class=\"timeline-body\"> <p>{{item.content}}</p> </div> </div> </li> </ul> ",
                styles: [".timeline { list-style: none; padding: 20px 0 20px; position: relative; } .timeline:before { top: 0; bottom: 0; position: absolute; content: \" \"; width: 2px; background-color: #e9ecf2; left: 25px; margin-right: -1.5px; } .timeline > li { margin-bottom: 20px; position: relative; box-sizing: border-box; } .timeline > li:before, .timeline > li:after { content: \" \"; display: table; } .timeline > li > .timeline-badge { color: #fff; width: 46px; height: 46px; line-height: 51px; font-size: 1.4em; text-align: center; position: absolute; top: 16px; left: 0px; margin-right: -25px; background-color: #999999; z-index: 100; border-radius: 9999px; } .timeline > li > .timeline-panel { width: calc( 100% - 65px); float: right; border: 1px solid #edeeed; border-radius: 2px; padding: 10px 20px; position: relative; box-shadow: none; } .timeline > li:after { clear: both; } "]
            },] },
];
/**
 * @nocollapse
 */
Ng2TimelineSimpleComponent.ctorParameters = function () { return []; };
Ng2TimelineSimpleComponent.propDecorators = {
    'items': [{ type: _angular_core.Input },],
};

var Ng2TimelineSimple = (function () {
    function Ng2TimelineSimple() {
    }
    return Ng2TimelineSimple;
}());
Ng2TimelineSimple.decorators = [
    { type: _angular_core.NgModule, args: [{
                imports: [
                    _angular_common.CommonModule
                ],
                declarations: [Ng2TimelineSimpleComponent],
                providers: [],
                exports: [Ng2TimelineSimpleComponent]
            },] },
];
/**
 * @nocollapse
 */
Ng2TimelineSimple.ctorParameters = function () { return []; };

exports.Ng2TimelineSimple = Ng2TimelineSimple;

Object.defineProperty(exports, '__esModule', { value: true });

})));
