// ng2-timeline-simple/dist/ng2timelineSimple.component.js
/**
 * Created by mohma on 7/31/2017.
 */
import { Component, Input } from '@angular/core';
var Ng2TimelineSimpleComponent = (function () {
    function Ng2TimelineSimpleComponent() {
    }
    /**
     * @return {?}
     */
    Ng2TimelineSimpleComponent.prototype.ngOnInit = function () {
    };
    return Ng2TimelineSimpleComponent;
}());
export { Ng2TimelineSimpleComponent };
Ng2TimelineSimpleComponent.decorators = [
    { type: Component, args: [{
                selector: 'ng2-timeline-simple',
                template: "<ul class=\"timeline\"> <li *ngFor=\"let item of items; index as i\"> <div class=\"timeline-badge\" [innerHTML]=\"item.icon\" [ngStyle]=\"{'background-color': item.color}\"> </div> <div class=\"timeline-panel\"> <div class=\"timeline-heading\"> <h4 class=\"timeline-title\">{{item.title}}</h4> </div> <div class=\"timeline-body\"> <p>{{item.content}}</p> </div> </div> </li> </ul> ",
                styles: [".timeline { list-style: none; padding: 20px 0 20px; position: relative; } .timeline:before { top: 0; bottom: 0; position: absolute; content: \" \"; width: 2px; background-color: #e9ecf2; left: 25px; margin-right: -1.5px; } .timeline > li { margin-bottom: 20px; position: relative; box-sizing: border-box; } .timeline > li:before, .timeline > li:after { content: \" \"; display: table; } .timeline > li > .timeline-badge { color: #fff; width: 46px; height: 46px; line-height: 51px; font-size: 1.4em; text-align: center; position: absolute; top: 16px; left: 0px; margin-right: -25px; background-color: #999999; z-index: 100; border-radius: 9999px; } .timeline > li > .timeline-panel { width: calc( 100% - 65px); float: right; border: 1px solid #edeeed; border-radius: 2px; padding: 10px 20px; position: relative; box-shadow: none; } .timeline > li:after { clear: both; } "]
            },] },
];
/**
 * @nocollapse
 */
Ng2TimelineSimpleComponent.ctorParameters = function () { return []; };
Ng2TimelineSimpleComponent.propDecorators = {
    'items': [{ type: Input },],
};
function Ng2TimelineSimpleComponent_tsickle_Closure_declarations() {
    /** @type {?} */
    Ng2TimelineSimpleComponent.decorators;
    /**
     * @nocollapse
     * @type {?}
     */
    Ng2TimelineSimpleComponent.ctorParameters;
    /** @type {?} */
    Ng2TimelineSimpleComponent.propDecorators;
    /** @type {?} */
    Ng2TimelineSimpleComponent.prototype.items;
}
